tests/HttpClientStub.php

```diff
@@ -32,9 +32,12 @@ public function setPostHandler(?callable $postHandler): void
         self::$postHandler = $postHandler;
     }
 
-    public function requestJson(Service $service, $requestUrl, $params)
-    {
-        if (self::$jsonHandler !== null) {
+    public function requestJson(
+        Service $service,
+        $requestUrl,
+        $params
+    ) {
```

Comment:
Missing return and parameters types

Reply:
This class implements Adyen/ClientInterface and there parameters are not typehinted so if i try to typehint them get an error.